boa/src/builtins/regexp/mod.rs

```diff
@@ -269,16 +404,33 @@ pub fn r#match(this: &Value, arg: String, ctx: &mut Interpreter) -> ResultValue
     }
 }
 
-/// Return a string representing the regular expression
+/// `RegExp.prototype.toString()`
+///
+/// Return a string representing the regular expression.
+///
+/// More information:
+///  - [ECMAScript reference][spec]
+///  - [MDN documentation][mdn]
+///
+/// [spec]: https://tc39.es/ecma262/#sec-regexp.prototype.tostring
+/// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/toString
 pub fn to_string(this: &Value, _: &[Value], _: &mut Interpreter) -> ResultValue {
     let body = from_value::<String>(this.get_internal_slot("OriginalSource")).map_err(to_value)?;
     let flags = this.with_internal_state_ref(|regex: &RegExp| regex.flags.clone());
     Ok(to_value(format!("/{}/{}", body, flags)))
 }
 
-/// RegExp.prototype[Symbol.matchAll]
-/// Returns all matches of the regular expression against a string
-/// TODO: it's returning an array, it should return an iterator
+/// `RegExp.prototype[ @@matchAll ]( string )`
+///
+/// The `[@@matchAll]` method returns all matches of the regular expression against a string.
+///
+/// More information:
+///  - [ECMAScript reference][spec]
+///  - [MDN documentation][mdn]
+///
+/// [spec]: https://tc39.es/ecma262/#sec-regexp-prototype-matchall
+/// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/@@matchAll
+// TODO: it's returning an array, it should return an iterator
```

Comment:
We should probably add issues to the repo with these things.

Reply:
Yep. I thought of sifting through the code to find the `// TODO:` and `// FIXME:` and creating issues for them.